net-libs/libsearpc/libsearpc-3.2.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python{2_7,3_5,3_6})
+
+inherit autotools python-single-r1
+
+DESCRIPTION="A simple C language RPC framework"
+HOMEPAGE="https://github.com/haiwen/libsearpc/ http://seafile.com/"
+SRC_URI="https://github.com/haiwen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}
+	>=dev-libs/glib-2.26.0
+	>=dev-libs/jansson-2.2.1
+	>=sys-devel/automake-1.16.1-r2:1.16"
```

Comment:
There's a better way to do this using autotools.eclass, I'll fix it in a separate commit. You should become a maintainer for this package though ;)

Reply:
Well, you can add me with your fix or I will do it in the next release.

Reply:
I took the information from this page https://wiki.gentoo.org/wiki/Project:Python/Byte_compiling Probably you could update it with proper autotools.eclass fix. Thanks!

Reply:
> Well, you can add me with your fix or I will do it in the next release.

Please do it next time :)